en/advanced_features/precland.md

```diff
@@ -70,29 +74,28 @@ If the beacon is still not visible at the search altitude and after a search tim
 In *Opportunistic Mode* the vehicle will use precision landing *if* (and only if) the beacon is visible when landing is initiated.
 If it is not visible the vehicle immediately performs a *normal* landing at the current position.
 
-## Performing a Precision Landing
+## Initiating a Precision Landing
 
 :::note
-Due to a limitation in the current implementation of the position controller, precision landing is only possible with a valid global position.
+Precision landing is only possible with a valid global position (due to a limitation in the current implementation of the position controller).
```

Comment:
1. I assume this is still true?
2. We're missing the option to do this in RTL using `RTL_PLD_MD` too right?

Reply:
1. Good point. Yes, this should still be the case, but might change in the future after we complete further refactoring of the navigator module.
2. Actually the RTL method using `RTL_PLD_MD` is the only method I know if. I have not seen a standalone "Precision Landing" flight mode as described in the docs.

Reply:
Just checked in the latest QGC and there I don't see the precision landing flight mode :sweat_smile: 
![image](https://user-images.githubusercontent.com/3762382/129006622-e7d60d26-63f3-41b6-a1d8-e0d03fd4b369.png)

Reply:
But then again `commander mode auto:precland` works just fine. For now I will focus on what I know and add this information. Perhaps I can figure out how the flight modes work later.

Reply:
The list of flight modes is hard coded, and indicates the set of modes that you can actually *set* - which is not true for precland. However if you put into this mode QGC will display it correctly (where it says "Ready to Fly" above).

You can set the modes using the MAV_CMD_SET_MODE command (I'm 99% sure) but this requires you to set the base and custom modes appropriately. This is probably not be as simple as setting [the values as 4 and 9](https://github.com/PX4/PX4-Autopilot/blob/master/src/modules/commander/px4_custom_mode.h#L66). It would be much better to abstract these by supporting precision landing in the landing commands.


Yes, and if you put into this mode you'll see